pkg/controllers/node/termination/terminator/eviction.go

```diff
@@ -182,11 +184,15 @@ func (q *Queue) Evict(ctx context.Context, key QueueKey) bool {
 		}); err != nil {
 		// status codes for the eviction API are defined here:
 		// https://kubernetes.io/docs/concepts/scheduling-eviction/api-eviction/#how-api-initiated-eviction-works
-		if apierrors.IsNotFound(err) || apierrors.IsConflict(err) {
+		if apierrors.IsNotFound(err) {
 			// 404 - The pod no longer exists
 			// https://github.com/kubernetes/kubernetes/blob/ad19beaa83363de89a7772f4d5af393b85ce5e61/pkg/registry/core/pod/storage/eviction.go#L160
+			return true
+		}
+		if apierrors.IsConflict(err) {
 			// 409 - The pod exists, but it is not the same pod that we initiated the eviction on
 			// https://github.com/kubernetes/kubernetes/blob/ad19beaa83363de89a7772f4d5af393b85ce5e61/pkg/registry/core/pod/storage/eviction.go#L318
+			metrics.NodesEvictionRequestsTotal.With(map[string]string{metrics.ResponseLabel: "conflict"}).Inc()
```

Comment:
Should we just consider surfacing the raw status codes and then you don't have to worry about creating custom labels yourself? `client_go_request_total` calls it `code`